### PR TITLE
Feature/detection rebase

### PR DESCRIPTION
**Pull request by hmightypirate**

### CODE REVIEW

File: src/hal/setup.c [hunk 1]
```diff
@@ -126,8 +128,16 @@ static void setup_clock(void)
  */
 static void setup_exceptions(void)
 {
-	nvic_enable_irq(NVIC_DMA2_STREAM7_IRQ);
-	nvic_enable_irq(NVIC_USART1_IRQ);
+
+  // setting priorities
+  // led emitters
+  nvic_set_priority(NVIC_TIM1_UP_TIM10_IRQ, 0);
```
Comment (eLeDe) [inline src/hal/setup.c:134]: We must set here all interruptions priorities

**Review by eLeDe — CHANGES_REQUESTED** [review]
Set interruption priorities to all enabled IRQs

**Comment by hmightypirate** [comment]
Done

**Review by eLeDe — APPROVED** [review]
:ok_hand:

**Comment by punkto** [comment]
:ok:

**Comment by hmightypirate** [comment]
Removing this pull because the conflict was resolved in another pull

**Pull request closed without merging**